core/kazoo_endpoint/src/kz_endpoint.erl

```diff
@@ -1562,6 +1563,22 @@ bowout_settings('false') ->
     ,{<<"Loopback-Bowout">>, <<"true">>}
     ].
 
+
+-spec maybe_set_call_forward_fail(ccv_acc()) -> ccv_acc().
+maybe_set_call_forward_fail({_Endpoint, _Call, 'undefined', _CCVs}=Acc) ->
+    Acc;
+maybe_set_call_forward_fail({Endpoint, Call, CallFwd, CCVs}=Acc) ->
+    case kapps_config:get_is_true(?CONFIG_CAT, <<"fail_on_single_call_forward_reject">>, 'false') of
+        'true' ->
+            Fail = kapps_config:get_binary(?CONFIG_CAT, <<"fail_on_reject_val">>, <<"NORMAL_CLEARING">>),
```

Comment:
@tsromana maybe you have to set `continue_on_fail` too? setting `fail_on_single_reject` would cause the bridge attempt to failed, and if the `fail_on_single_reject` has not been set the call is terminated.